synapse/storage/devices.py

```diff
@@ -127,16 +129,21 @@ def get_devices_by_remote(self, destination, from_stream_id, limit):
         # (user_id, device_id) entries into a map, with the value being
         # the max stream_id across each set of duplicate entries
         #
-        # maps (user_id, device_id) -> stream_id
+        # maps (user_id, device_id) -> (stream_id, context)
         # as long as their stream_id does not match that of the last row
```

Comment:
I don't really know what this line means. I think it should go away. @anoadragon453  ?